schemas/test/en/test_list_collector_repeating_blocks.json

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
I am not sure if these schemas should be so abstract if we modelled them after prototypes that were based on real life surveys in all previous iterations of looping.

Reply:
Yeah I agree, I regretted making it the way I did afterwards. I had been using `test_list_collector_repeating_blcoks_section_summary` as a the functional and manual testing schema, and made this one for unit test purposes. 
Won't do it again!

Reply:
I see, do we need both schemas? Can we not do all the testing with only one and one for the hub?

Reply:
We had to investigate and delete duplicated/bloated schemas before, so just want to avoid that in future.

Reply:
Understand, will change the schema to be less abstract instead of removing as I think it's still beneficial to have a schema of only the list collector with RB + means not having to alter tests as much and getting this merged quicker to unblock

Reply:
Added context to schema to make it less abstract 👍

Reply:
What is the difference between these two schemas except extra question block after list collector? If it's duplicated schema we shouldn't keep both if they are essentially the same.

Reply:
The difference is one has a section summary and one doesn't

Reply:
I'd be in favour of keeping it - seems like a good idea to have a really simple one that isolates the new feature